src/main/kotlin/blackjack/domain/Score.kt

```diff
@@ -1,10 +1,28 @@
 package blackjack.domain
 
-class Score(private val denominations: List<Denomination>) {
-    fun calculate(): Int {
+import kotlin.math.abs
+
+class Score(
+    private val cards: Cards
+) {
+    val value: Int = toScore()
```

Comment:
cards를 프로퍼티로 들고 있을 필요가 있을지 고민해보아요!

```suggestion
class Score(
    cards: Cards
) {
    val value: Int = toScore()
    val cardSzie: Int = cards.size
```